devops/azure-pipelines.yml

```diff
@@ -46,16 +48,18 @@ stages:
     steps:
     - checkout: self
     - checkout: templates
-    - template: ./templates/analysis.yml@templates
-      parameters:
-        microservice: tracking-microservice
-        workDir: ./project-registry-tracking-microservice/
-        sonarKeyPrefix: $(sonarKeyPrefix)
-        sonarNamePrefix: $(sonarNamePrefix)
-        sonarOrg: $(sonarOrg)
-        sonarUrlPrefix: $(sonarUrlPrefix)
-        discordChannel: $(discordAnalysisChannel)
-        discordKey: $(discordAnalysisChannel)
+    # - template: ./templates/analysis.yml@templates
+    #   parameters:
+    #     microservice: tracking-microservice
+    #     workDir: ./project-registry-tracking-microservice/
+    #     sonarKeyPrefix: $(sonarKeyPrefix)
+    #     sonarNamePrefix: $(sonarNamePrefix)
+    #     sonarOrg: $(sonarOrg)
+    #     sonarUrlPrefix: $(sonarUrlPrefix)
+    #     discordChannel: $(discordAnalysisChannel)
+    #     discordKey: $(discordAnalysisKey)
+
+    - script: echo $(Build.Repository.Uri)
```

Comment:
this doesn't look ready for review

Reply:
Oops, having several nearly identical YAML files made this get overlooked.  Sorry.
Anyway, it is fixed now.